lib/rmt/lockfile.rb

```diff
@@ -4,13 +4,13 @@ class RMT::Lockfile
 
   class << self
     def create_file
-      if File.exist?(LOCKFILE_LOCATION)
-        raise RMT::Lockfile::ExecutionLockedError
-      else
-        dirname = File.dirname(LOCKFILE_LOCATION)
-        FileUtils.mkdir_p(dirname) unless File.directory?(dirname)
-        File.open(LOCKFILE_LOCATION, 'w') { |f| f.write(Process.pid) }
-      end
+      dirname = File.dirname(LOCKFILE_LOCATION)
```

Comment:
Why so complicated?
Why a relative path in `LOCKFILE_LOCATION`? Lockfiles have to go to `/var/lock` which nowadays in most distributions (including openSUSE and SLES) is just a link to `/run/lock`. Please double-check if I'm right. Where does for example zypper put its lockfile? But I'm sure you can make the code simpler without the `../../`